rvv-intrinsic-rfc.md

```diff
@@ -348,70 +348,83 @@ vmerge.vvm vd, vs2, vs1, v0:
 vint8m1_t vmerge_vvm_i8m1_m(vbool8_t mask, vint8m1_t vs2, vint8m1_t vs1, size_t vl);
 ```
 
-## Tail Policy in the Intrinsics<a name="tail-policy"></a>
+## Policy Intrinsic Functions<a name="policy"></a>
 
-The naming rule of intrinsics with tail policy is
+Users could use policy intrinsic functions to control tail and mask policy.
+
+Enocde the policy explicitly in function name suffix if operations support those policies.
 
 ```
-INTRINSIC_WITH_ADDITIONAL_DEST ::= INTRINSIC '_tu'
+INTRINSIC_WITH_TU ::= INTRINSIC '_tu'
+INTRINSIC_WITH_TA ::= INTRINSIC '_ta'
+INTRINSIC_WITH_MASK_TU_MU ::= INTRINSIC '_tumu'
+INTRINSIC_WITH_MASK_TA_MU ::= INTRINSIC '_tamu'
+INTRINSIC_WITH_MASK_TU_MA ::= INTRINSIC '_tuma'
+INTRINSIC_WITH_MASK_TA_MA ::= INTRINSIC '_tama'
 ```
 
 ```
 Example:
 
-vadd.vv vd, vs2, vs1, v0.t:
+vadd.vv vd, vs2, vs1:
 
-// Add an additional `dest` argument to control tail undisturbed/agnostic.
-// `dest` == vundefined(), tail agnostic.
-vint8m1_t vmv_v_x_i8m1_tu(vint8m1_t dest, int8_t src, size_t vl);
+vint8mf8_t vadd_vv_i8mf8_tu (vint8mf8_t undisturbed, vint8mf8_t op1, vint8mf8_t op2, size_t vl);
 ```
 
-Define two constants for tail policy. Users could use these two constants for `ta` argument.
-```
-#define VE_TAIL_UNDISTURBED 0
-#define VE_TAIL_AGNOSTIC 1
-```
+For intrinsics without the destination argument which may use tail and mask value.
 
-For intrinsics with `maskedoff` and `ta` argument:
-| Masked? | Needs tail preserved | Needs maskedoff preserved | Intrinsic
-| ------- | -------------------- | ------------------------- | ------------------------------------------------------------------------
-| No      | No                   | N/A                       | `vadd_vv_<ty>(vs2, vs1, vl)`
-| No      | Yes                  | N/A                       | `vadd_vv_<ty>_tu(dest, vs2, vs1, vl)`
-| Yes     | No                   | No                        | No support.
-| Yes     | No                   | Yes                       | No support.
-| Yes     | Yes                  | Yes                       | No support.
-| Yes     | Yes                  | No                        | No support. Tail undisturbed and maskedoff agnostic is likely rare.
+Masked?  | TU? | MU? | Intrinsic
+--- |--- |--- |---
+No  | No | N/A| `vadd_vv_i8m1_ta(vs2, vs1, vl)` and `vadd_vv_i8m1(vs2, vs1, vl)`
+No  | Yes| N/A| `vadd_vv_i8m1_tu(undisturbed,vs2, vs1, vl)`
+Yes | No | No | `vadd_vv_i8m1_tama(mask, vs2, vs1, vl)`
+Yes | No | Yes| `vadd_vv_i8m1_tamu(mask, merge, vs2, vs1, vl)` and `vadd_vv_i8m1_m(mask, merge, vs2, vs1, vl)` if merge is `vundefined()`
+Yes | Yes| No | `vadd_vv_i8m1_tuma(mask, merge, vs2, vs1, vl)`
```

Comment:
I'm not clear why you call it `undisturbed` in the `tu` case, and `merge` in the `tuma` case. Suggest using same thing everywhere.